File: app/src/main/java/com/kanzankazu/itungitungan/view/util/HtmlUtil.java
package com.kanzankazu.itungitungan.view.util;

import android.text.TextUtils;

public class HtmlUtil {
    public static String checkBr(String sHtml) {
        String[] split = sHtml.split("<br>");
        if (split.length > 1) {
            if (TextUtils.isEmpty(split[1])) {
                return sHtml.replace("<br>", "\n");
            } else {
                return sHtml.replace("<br>", "");
            }
        }
        return sHtml;
    }
}

File: app/src/main/java/com/kanzankazu/itungitungan/util/HtmlUtil.kt
package com.kanzankazu.itungitungan.util

import android.text.TextUtils

object HtmlUtil {
    fun checkBr(sHtml: String): String {
        val split = sHtml.split("<br>".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
        return if (split.size > 1) {
            if (TextUtils.isEmpty(split[1])) {
                sHtml.replace("<br>", "\n")
            } else {
                sHtml.replace("<br>", "")
            }
        } else sHtml
    }
}
